File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
            zero_index = 0
            for i in range(len(nums)):
                if nums[i]!=0:
                    nums[i],nums[zero_index] = nums[zero_index], nums[i]
                    zero_index +=1

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int left  = 0;
        int right = 0;
        while(right < nums.length){
            if(nums[right] != 0){
                int tem = nums[right];
                nums[right] = nums[left];
                nums[left] = tem;
                left++;
            }
            right++;
        }
    }
}